pkg/asset/templates/content/manifests/cluster-mco-config.go

```diff
@@ -0,0 +1,65 @@
+package manifests
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+
+	"github.com/openshift/installer/pkg/asset"
+	"github.com/openshift/installer/pkg/asset/templates/content"
+)
+
+const (
+	mcoConfigTemplateFileName = "cluster-mco-02-config.yaml.template"
```

Comment:
Where does the eventual generated manifest's filename sort vs. [the installer-generated MachineSet manifests][1]?  Do we want a `90_` prefix or something to ensure we sort the MachineConfiguration first, so the MCO knows to leave MachineSets alone before the MachineSets are created in the cluster?

[1]: https://github.com/openshift/installer/blob/55b35185c79710706be350d384fc35c2afb322c3/pkg/asset/machines/worker.go#L71-L72